src/tests/sorting.spec.js

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '../fixtures/base';
+
+test.describe('sorting invertory page', () => {
+    test('sorting name A-Z', async ({ page, app }) => {
+        await app.login.navigate('/');
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(page).toHaveURL(app.inventory.url);
+
+        await app.inventory.sortBy('az');
+
+        await expect(app.inventory.inventoryItems.nth(0).getByTestId('inventory-item-name')).toHaveText('Sauce Labs Backpack');
+    });
+
+    test('sorting name Z-A', async ({ page, app }) => {
+        await app.login.navigate('/');
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(page).toHaveURL(app.inventory.url);
+
+        await app.inventory.sortBy('za');
+
+        await expect(app.inventory.inventoryItems.nth(0).getByTestId('inventory-item-name')).toHaveText('Test.allTheThings() T-Shirt (Red)');
+    });
+
+    test('sorting Price (low to high)', async ({ page, app }) => {
+        await app.login.navigate('/');
```

Comment:
Do you really need to login every time in tests?
If yes, do it in the beforeEach hook 

If not, remove it from tests to the beforeAll hook. (If you are worried about parallelism, then playwright uses one worker for the one file(with all tests), so there is no need to worry)